src/database/migrations/V1__addTest.ts

```diff
@@ -0,0 +1,19 @@
+import knex from 'knex';
+
+const TEST_TABLE = 'test';
+
+export async function up(db: knex) {
+  await db.schema
+    .createTable(TEST_TABLE, (table) => {
+      table.uuid('id').primary();
+      table.uuid('user_id').notNullable();
+      table.uuid('test_type_id').notNullable();
+      table.timestamp('creation_time').notNullable().index();
+      table.jsonb('results').nullable();
```

Comment:
Think we need to add tester_user_id as well, so we know who performed the test or added the results.